### ISSUE TITLE
Initial step for TensorBoard Callback

### ISSUE THREAD

**Issue by BananaLoaf** [tensorflow/tensorflow#45274]
**System information**
- TensorFlow version (you are using): 2.3.1
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
Add ```initial_train_step``` and ```initial_test_step``` as ```__init__``` arguments.
Currently, TensorBoard callback starts writing metrics from step 0 for both train and test, but it breaks the graphs if model.fit() was used in tandem with loaded checkpoint. *STEP* horizontal axis does not provide graph consistency, *RELATIVE* and *WALL* horizontal axis provides metrics consistency, but horizontal axis has huge value gaps in it. Each subsequent run starts writing from step 0

![image](https://user-images.githubusercontent.com/42458049/100635373-81e39a80-3341-11eb-8082-13c2d966e468.png)
*STEP horizontal axis*

![image](https://user-images.githubusercontent.com/42458049/100635480-a0499600-3341-11eb-85c8-3834de3759b5.png)
*RELATIVE horizontal axis, first small gap is from testing after epoch*

**Will this change the current api? How?**
No

**Who will benefit with this feature?**
People who use model.fit() with checkpoints

**Comment by stephanwlee** [comment]
This is a Keras callback bug which is owned by Kersas team.

**Comment by BananaLoaf** [comment]
> This is a Keras callback bug which is owned by Keras team.

To be fair,

**Comment by BananaLoaf** [comment]
That was an accident

**Comment by mawaregetsuka** [comment]
@BananaLoaf 
Looks like you used the model.fit() twice in your example, then you save the log to a same file. In fact, nobody knows except yourself that the two training are continuous and consistent. Sometimes it is possible to train the same model in different directions after a checkpoint.
So in your case all you need is the initial_epoch parameter of the model.fit().
https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit
That's my opinion and I hope it helps you.

**Comment by mohantym** [comment]
Hi @BananaLoaf !
We are checking to see whether you still need help in this issue . Did you check with above [suggestion ](https://github.com/tensorflow/tensorflow/issues/45274#issuecomment-740495910) yet?. Thanks!
